gherkin/ruby/spec/gherkin/messages/subprocess_cucumber_messages_spec.rb

```diff
@@ -6,13 +6,11 @@ module Messages
     describe SubprocessCucumberMessages do
```

Comment:
Can you rename this class to `Gherkin` to bring it in line with the Java implementation? It makes for a nicer API.

Reply:
Hey, @enkessler, if you want this, you've got it. Let me know otherwise and I'll grab it. :)

Reply:
Actually, @enkessler we're thinking about naming it differently so belay last for now. Sorry!

Reply:
Since I've merged this to master now, let's just do this on master. First come first serve!

Reply:
TODO: 

```
Let’s move `Gherkin::Messages::SubprocessCucumberMessages` to `Gherkin::Gherkin`
and `Gherkin::Messages::ProtobufCucumberMessages` to`Gherkin::ProtobufCucumberMessages`. 
We don’t need the `Messages` module in the middle.
```

Reply:
@enkessler said he'll take a shot at it tonight.